src/PatientChecking/PatientChecking/Views/ViewModels/IPagination.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientChecking.Views.ViewModels
+{
+    interface IPagination
+    {
+        public int PageIndex { get; set; }
```

Comment:
Should remove the public in interface

Reply:
The public access modifier has been removed in interface.